database/things/lean.pldb

```diff
@@ -2,6 +2,7 @@ title Lean
 appeared 2015
 type pl
 website http://leanprover.github.io/
+originCommunity Microsoft Research
 domainName leanprover.github.io
```

Comment:
country United States
Leonardo de Moura is also based in Redmond, Washington.
https://www.microsoft.com/en-us/research/people/leonardo/